scheduler/plugin/host_test.go

```diff
@@ -96,39 +95,31 @@ func initKlog(t *testing.T, buf *bytes.Buffer) {
 	klog.SetOutput(buf)
 }
 
-func Test_k8sHandleEventRecorderEventFn(t *testing.T) {
```

Comment:
why is this test removed?

Reply:
Thank you for your comment.
I was checking the behavior of test code in `scheduler/plugin/host_test.go` and forgot to restore the `Test_k8sHandleEventRecorderEventFn`